src/api/getProducts.ts

```diff
@@ -0,0 +1,14 @@
+import { ProductSchema } from "@/schemas/ProductSchema";
```

Comment:
For type declarations, we usually place them as close as possible to their 'source of truth' as possible. In this case, `ProductSchema` is a type defined by the backend, so we would just declare it here in the api that fetches it, instead of having its own file, since it is a bit more practical.